rule_schema.yaml

```diff
@@ -0,0 +1,661 @@
+$id: https://raw.githubusercontent.com/returntocorp/semgrep/develop/cli/src/semgrep/rule_schema.yaml
+$schema: http://json-schema.org/draft-07/schema#
+$comment: "!!if you modify this file, you probably also need to update the one in semgrep-app:server/semgrep_app/rule_schema.yaml!!"
```

Comment:
this comment is still valid?

Reply:
Until I PR to use this via the submodule it will be. I can just update it now though.